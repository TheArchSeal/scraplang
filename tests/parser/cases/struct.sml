struct s {}
struct s { x: a, y: b = 0 }
