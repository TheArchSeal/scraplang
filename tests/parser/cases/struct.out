stmt (12):1:1 struct s
stmt (12):2:1 struct s
    member  :2:12 x
    type (3):2:15 a
    expr (1):2:12 (empty)
    member  :2:18 y
    type (3):2:21 b
    expr (3):2:25 0
